config.example.yml

```diff
@@ -12,7 +12,7 @@
     ENABLE_TSL: True
     TSL: -4
     TTP: 2
-   LOGGING:
```

Comment:
I still don't know why my IDE took away this whitespace.  But I'm glad it's getting reverted.